Form/ChoiceList/MongoDBQueryBuilderLoader.php

```diff
@@ -56,12 +56,8 @@ public function getEntities(): array
         return array_values($this->queryBuilder->getQuery()->execute()->toArray());
     }
 
-    /**
-     * @param string $identifier
-     *
-     * @return object[]
-     */
-    public function getEntitiesByIds($identifier, array $values): array
+    /** @return object[] */
+    public function getEntitiesByIds(string $identifier, array $values): array
```

Comment:
[`EntityLoaderInterface::getEntitiesByIds()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bridge/Doctrine/Form/ChoiceList/EntityLoaderInterface.php#L33)